Examples/Python/src/Geant4Component.cpp

```diff
@@ -109,53 +135,53 @@ PYBIND11_MODULE(ActsPythonBindingsGeant4, mod) {
                   cfg, getDefaultLogger("SensitiveSurfaceMapper", level));
             }));
 
+    py::class_<State>(sm, "State").def(py::init<>());
```

Comment:
why do we need to wire the state out?

Reply:
Hmm its helps testing the surface mapping outside of the simulation, which was convenient for debugging